### DISCUSSION TITLE
LCD Static/Scanlines & offset

### DISCUSSION THREAD

**Discussion by demetriusburgess** [Bodmer/TFT_eSPI#2262] [General]
Anybody have any idea why I'm having this issue? 

I was working with the tft_eSPI library & esp_now. Everything was working fine, then I reuploaded the sketch and started getting this weird noise at the top of the LCD. Additionally when I use tft.fillScreen(TFT_BLACK); the screen remains white. I've tried other colors like TFT_BLUE, which seems to work fine (maybe a little washed out). I'm thinking the screen it's self is just broken, but I'm also noticing that every sketch displayed on the screen is cropped and offset about the same amount of space that is distorted. Again I'm not sure if this is an actual "Issue" or just a broken display. Any insight would be helpful and greatly appreciated.  

![Screenshot_20221220_034739](https://user-images.githubusercontent.com/7832292/208763992-d779596d-9c58-4c52-97cc-a8836fe61d88.png)
![Screenshot_20221220_035002](https://user-images.githubusercontent.com/7832292/208763998-48e6f89a-64d2-4ceb-a659-8ddc5deca80a.png)
![Screenshot_20221220_035015](https://user-images.githubusercontent.com/7832292/208764000-5a4efb2d-f656-4287-a495-dcb15d745bd9.png)
![Screenshot_20221220_035059](https://user-images.githubusercontent.com/7832292/208764003-817a8b02-9d6c-4010-9d6b-1f4b3e54bf15.png)

**Comment by Bodmer** [comment]
Maybe you updated the library and your setup got over-written?

Run the Read_User_Setup sketch and check the serial monitor output.

Check the right size screen is specified in your setup and offsets enabled, example for a TTGO_T_Display here:

https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup25_TTGO_T_Display.h

**Reply by demetriusburgess** [reply]
Thank you! Yo were right, that seems to have been exactly what happened. The issue was that #include <User_Setup.h> was not commented out after the update. It was actually the same exact issue as this issue from a year ago https://github.com/Bodmer/TFT_eSPI/issues/1484 

Everything's working great now. You're amazing! 

![Screenshot_20221221_014648](https://user-images.githubusercontent.com/7832292/208839731-4fc144ec-ba58-417e-97ae-7cfd0e74f3f4.png)
![Screenshot_20221221_014707](https://user-images.githubusercontent.com/7832292/208839733-f774bf6f-0e6d-4f57-a417-a6784ce548fe.png)
![Screenshot_20221221_014737](https://user-images.githubusercontent.com/7832292/208839735-20bc48ce-c134-480f-86af-0ea4e24f8703.png)

**Reply by AmberElferink** [reply]
Thank you for updating it with the issue of #include <User_Setup.h> not being commented out!!! This was my issue as well!

**Reply by ImNewPlsHelpMe** [reply]
![Uploading IMG_20250320_022917_354.jpg…]()
![Uploading IMG_20250319_155437_197.jpg…]()

![Uploading IMG_20250322_044223_892.jpg…]()
![Uploading IMG_20250320_232409_448.jpg…]()
can some explain me like clearly, I'm new at coding and using tft, just for our project, can someone tell me if we have the same issue? thank you for ur reply in advance
